x/datadeal/types/message_deal_test.go

```diff
@@ -0,0 +1,166 @@
+package types
```

Comment:
Does has the reason to write this test code?

Reply:
Because I wanted to do a single test for `ValidateBasic()`